### DISCUSSION TITLE
Broken links in documentation

### DISCUSSION THREAD

**Discussion by maurobalbi** [dotansimha/graphql-code-generator#6604] [General] [answered]
I just noticed that the links on [https://www.graphql-code-generator.com/docs/custom-codegen/write-your-plugin](https://www.graphql-code-generator.com/docs/custom-codegen/write-your-plugin) pointing to the APIs are broken. It seems that grapqhl-js doesn't include the *.d.ts' in the repo anymore causing the links to go to 404. 

If you have any pointers to where I can find the most recent type definitions for the schema and documents, I'd be happy to open a PR.

**Comment by dotansimha** [comment, marked as answer]
Thanks! Fixed, will be available in the next deployment.
